src/Features/Core/Portable/CodeRefactorings/CodeRefactoringService.cs

```diff
@@ -289,7 +289,7 @@ private ImmutableArray<CodeRefactoringProvider> CreateRefactorings(string langua
                                         attribute.Languages.Length == 0 ||
                                         attribute.Languages.Contains(language))
                                     {
-                                        builder.Add((CodeRefactoringProvider)Activator.CreateInstance(typeInfo.AsType()));
+                                        builder.Add((CodeRefactoringProvider)Activator.CreateInstance(typeInfo.AsType())!);
```

Comment:
How can Activator.CreateInstance() return null? I would have presumed it can't since the whole point of it is to create things....do we have a bug here?

Reply:
`Activator.CreateInstance(typeof(int?))` 😢